src/olympia/addons/models.py

```diff
@@ -1652,18 +1675,23 @@ def auto_approval_delayed_until_unlisted(self):
 
     @property
     def auto_approval_delayed_indefinitely(self):
-        return (
-            self.auto_approval_delayed_until == datetime.max
-            or self.auto_approval_delayed_until_unlisted == datetime.max
-        )
+        return self.auto_approval_delayed_until == datetime.max
 
     @property
     def auto_approval_delayed_temporarily(self):
         return (
             bool(self.auto_approval_delayed_until)
             and self.auto_approval_delayed_until != datetime.max
             and self.auto_approval_delayed_until > datetime.now()
-        ) or (
+        )
+
+    @property
+    def auto_approval_delayed_indefinitely_unlisted(self):
+        return self.auto_approval_delayed_until_unlisted == datetime.max
+
+    @property
+    def auto_approval_delayed_temporarily_unlisted(self):
```

Comment:
Random thought for another time - we should consider changing this kind of `@property`s to a function that takes a channel parameter so we don't need duplicates when we have additional channels than listed and unlisted.

Reply:
It's a little annoying because it's directly a "flag" for reviewers `get_flag()` function, which generates css classes...